sdk/src/main/java/com/uid2/UID2Manager.kt

```diff
@@ -487,16 +485,15 @@ public class UID2Manager internal constructor(
         @Throws(InitializationException::class)
         public fun init(
             context: Context,
+            serverUrl: String = UID2_API_URL_DEFAULT,
```

Comment:
I don't feel strongly on the name of this parameter, so copied what @sunnywu used in the developer docs